app/assets/javascripts/services/miq_service.js

```diff
@@ -158,4 +158,20 @@ ManageIQ.angular.app.service('miqService', ['$timeout', '$document', '$q', 'API'
       miqService.sparkleOff();
     }
   };
+
+  this.redirectBack = function(message, flashType, redirectUrl, reload) {
+    var flash = { message: message };
+
+    flash.level = flashType;
+
+    miqFlashLater(flash);
+
+    $timeout(function() {
+      $window.location.href = redirectUrl;
+
+      if(reload) {
```

Comment:
This looks broken, in the best case, nothing gets run after `location.href = ...`, so this is dead code.

In the worst case, the browser will start changing the URL, then get the reload and start reloading the current page instead.

Since this is a new thing, was there a reason to add it?